### DISCUSSION TITLE
Export annotations to a new pdf

### DISCUSSION THREAD

**Discussion by abul4fia** [RyotaUshio/obsidian-pdf-plus#283] [Q&A] [answered]
Hello!

I really like the approach of keeping my annotations in separate markdown files rather than embedding them directly in the PDF. The workflow that PDF++ offers is fantastic because it allows me to organize my notes across multiple markdown files while still being able to view them all in the PDF, without modifying the file itself. It's brilliant.

However, when I want to share my notes with colleagues who don’t use Obsidian, it would be more convenient to embed all those annotations into the PDF and share the annotated PDF directly. Is this possible?

**Comment by bbsmashdown** [comment]
I would also love to see this feature. As much as I hope obsidian (and PDF++) remain a stable part of my workflow, I have already seen a prior app/method of note taking fail - when I was in school (a decade ago now) and using the ipad to annotate lecture slides, I used an app that I no longer have access to, but because everything was able to be embedded in the pdf I can still review them on any platform with a pdf reader. Sorry I’m of no technical help, but just wanted to echo the desire for this feature.  

Thanks for the great plugin as is! Amazing work.

**Comment by joshvictor1024** [comment]
Would also like to echo this. I have been using Zotero + Obsidian for 2 years, and so far I've made annotations in Zotero and used Zotero item links (`zotero://...`, accessible through e.g., Zutilo) in Obsidian to refer back to the PDF files. Thanks to this wonderful plugin, I no longer have this back-and-forth hassle.

In Zotero, `File > Save as` exports the PDF with annotations, which I can share directly, echoing OP's use case. It'd be convenient if PDF++ had a similar function.

**Comment by Eric-iSt101** [comment]
I would like to have this feature implemented as well! Thank you again for the plugin!

**Comment by jldiaz** [comment, marked as answer]
Hi everyone,

Since I needed this exact feature for my own workflow, I ended up writing an Obsidian plugin to solve it: [PDF++ Exporter](https://github.com/jldiaz/pdfplus-exporter) (also available directly in the Obsidian Community Plugins directory).
It scans your Markdown backlinks, extracts your comments/callouts, and reconstructs native PDF highlight and popup annotations using pdf-lib.

If you'd like to give it a try, I'd really appreciate your feedback! Feel free to report any issues, edge cases, or feature requests over at the [PDF++ Exporter issues page](https://github.com/jldiaz/pdfplus-exporter/issues).

Hope it helps!
